lib-core/src/DiagramEngine.ts

```diff
@@ -71,18 +62,11 @@ export class DiagramEngine extends BaseEntity<DiagramEngineListener> {
 
 			//pop it onto the window so our E2E helpers can find it
 			if (window) {
-				(window as any)["diagram_instance"] = this;
+				(window as any)['diagram_instance'] = this;
```

Comment:
You can add `process.env.NODE_ENV === "development"`, so when production build it will not be included.

Reply:
we dont actually need this anymore :D